README.md

```diff
@@ -214,8 +214,8 @@ Example:
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/214270606-bac64c45-c1bf-4e39-a967-7fcacd4ca088.png)

**[Optional]**

- [ ] Your commit messages are not capitalized. It would be nice if you make them capitalize to make your pull request  professional :+1:

Reply:
Done as advised